corpora-api.yml

```diff
@@ -492,7 +494,152 @@ components:
       type: string
     visibility:
       type: string
-      enum: [public,private]
+      enum: [PUBLIC, PRIVATE]
+    project:
+      type: object
+      properties:
+        description:
+          type: string
+        name:
+          type: string
+        id:
+          $ref: '#/components/schemas/project_uuid'
+        s3_bucket_key:
+          type: string
+        owner:
+          type: object
+          properties:
+            id:
+              $ref: '#/components/schemas/user_uuid'
+            email:
+              type: string
+        status:
+          type: string
+          enum: [LIVE, EDIT]
+        processing_state:
+          type: string
+          enum: [NA, IN_VALIDATON, IN_ARTIFACT_CREATION, IN_DEPLOYMENT]
+        validation_state:
+          type: string
+          enum: [NOT_VALIDATION, VALID, INVALID]
+        links:
+          type: array
+          items:
+            type: object
+            properties:
+              url:
+                type: string
+              type:
+                type: string
+                enum: [PROTOCOL, RAW_DATA, OTHER]
+        attestation:
+          type: object
+          properties:
+            needed:
+              type: boolean
+            url:
```

Comment:
Given that tc_uri is below, the url isn't needed here. I'm good either way though: either not have the `attestation` object and just have two properties (`attestation_needed` and `tc_uri`) or have an `attestation` object containing a boolean representing whether it's required and the tc_uri.